qsirecon/workflows/recon/utils.py

```diff
@@ -82,3 +84,54 @@ def init_discard_repeated_samples_wf(
     ])  # fmt:skip
 
     return workflow
+
+
+def init_scalar_output_wf(
+    name="scalar_output_wf",
+):
+    """Write out reconstructed scalar maps."""
+    inputnode = pe.Node(
+        niu.IdentityInterface(
+            fields=[
+                "source_file",
+                "scalar_configs",
+            ],
+        ),
+        name="inputnode",
+    )
+    outputnode = pe.Node(
+        niu.IdentityInterface(fields=["scalar_files"]),
+        name="outputnode",
+    )
+    workflow = pe.Workflow(name=name)
+
+    organize_scalar_data = pe.MapNode(
+        OrganizeScalarData(),
```

Comment:
so this is how you get the data added to sidecars?

Reply:
It splits up the scalar dictionary object into individual fields that the DerivativesDataSink can work with, including individual entities that go into the filename and the metadata that goes into the sidecar.

If we want to patch in other metadata (e.g., `Sources` per #69) then we'll need to merge the metadata dictionary from `OrganizeScalarData` with whatever those metadata are before patching them into the DerivativesDataSink.